layers/+spacemacs/spacemacs-defaults/funcs.el

```diff
@@ -863,7 +863,11 @@ ones created by `magit' and `dired'."
 (defun spacemacs/copy-file-path ()
   "Copy and show the file path of the current buffer."
```

Comment:
Please update the docstring to reflect the change in behavior.

Reply:
updated